aixplain/modules/model/__init__.py

```diff
@@ -58,7 +58,7 @@ class Model(Asset):
     def __init__(
         self,
         id: Text,
-        name: Text,
+        name: Text = "",
```

Comment:
Name can not be an empty string. Are we doing some kind of validation to avoid this?